src/charm.py

```diff
@@ -369,6 +420,11 @@ def _on_database_created(self, event: DatabaseCreatedEvent) -> None:
             logger.info("Hydra service is absent. Deferring the event.")
             return
 
+        if not self._log_level_valid:
+            logger.info(f"Invalid configuration value for log_level: {self._log_level}")
+            self.unit.status = BlockedStatus("Bad configuration value for log_level")
```

Comment:
I would reformat the status message a little
```suggestion
            self.unit.status = BlockedStatus("Invalid configuration value for log_level")
```